app/db/interface.py

```diff
@@ -66,5 +67,27 @@ def get_payments(self, payment_ids=None, payment_attributes=None):
         :param payment_attributes: <dict> a collection of payment attributes to be used in
                                    filtering for specific payments
         """
+        if payment_ids != None:
+            payments = self.db.session.query(Payment).filter(Payment.id.in_(payment_ids)).all()
```

Comment:
cool. tinkered with this for a minute, but couldn't get it working. plus, honestly, didn't want to rewrite tests for it at this point.

definitely something for next sprint, but don't want to spend time on it right now